oldnyc/geocode/diff_geojson.py

```diff
@@ -35,7 +34,7 @@ def features_to_geojson_file(features, filename):
 def calculate_distance_delta_for_image_id(a, b):
     assert a["type"] == "Point"
     assert b["type"] == "Point"
-    (a_lng, a_lat) = b["coordinates"]
+    (a_lng, a_lat) = a["coordinates"]
```

Comment:
This doozy was introduced in #127. It's not in the OldTO version, so this one's on me!